heron/explorer/src/python/main.py

```diff
@@ -28,7 +28,7 @@
 import heron.explorer.src.python.topologies as topologies
 import heron.common.src.python.utils.config as config
 import heron.explorer.src.python.version as version
-from heron.common.src.python.color import Log
+import heron.common.src.python.utils.log as log
```

Comment:
`log` module has a logger object `Log` and a function `configure`. I first get the logger object out (see: https://github.com/twitter/heron/pull/1204/files#diff-762d509cb715b724772c61e8aae7f089R33) and then call `log.configure` under ``main`` function.